client_admin/pulp/client/admin/tasks.py

```diff
@@ -162,6 +175,33 @@ def cancel(self, **kwargs):
             else:
                 raise
 
+    def purge(self, **kwargs):
```

Comment:
Seems like due to the fact that this method is in BaseTasksSection other classes inherit it and it appears in 'repo tasks' section.
Right now we can list tasks related to one specific repo. If you want to leave the 'purge' in this section - we should be able also to remove tasks of specific repo. Otherwise I think 'purge' should not be present here.
@mhrivnak  what is your opinion on this?
```
$ pulp-admin repo tasks
Usage: pulp-admin [SUB_SECTION, ..] COMMAND
Description: list and cancel tasks related to a specific repository

Available Commands:
  cancel  - cancel one or more tasks
  details - displays more detailed information about a specific task
  list    - lists tasks queued (waiting) or running on the server
  purge   - purge tasks in one or more completed states
[ipanova@ina pulp]$ 
[ipanova@ina pulp]$ pulp-admin repo tasks list
Command: list
Description: lists tasks queued (waiting) or running on the server

Available Arguments:

  --repo-id - (required) identifies the repository to display
The following options are required but were not specified:
  --repo-id
[ipanova@ina pulp]$ pulp-admin repo tasks purge

+----------------------------------------------------------------------+
                         Purge Completed Tasks
+----------------------------------------------------------------------+

Command: purge
Description: purge tasks in one or more completed states

Available Arguments:

  --all, -a   - if specified, all tasks in "finished, skipped and error" states
                can be purged
  --state, -s - comma-separated list of tasks states desired to be purged.
                Example: "finished,error". Do not include spaces.
```